webrtc.html

```diff
@@ -2849,6 +2849,16 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       Each track to be sent is associated with exactly one <code><a>RTCRtpSender</a></code>, and
       each track to be received is associated with exactly one <code><a>RTCRtpReceiver</a></code>.</p>
 
+      <p>The basic principle is that the encoding and transmission of each
+      <code>MediaStreamTrack</code> is made such that the characteristics are as far as possible
+      maintained by the track created on the remote side. In other words, the
```

Comment:
This sentence structure confused me. I started reading "the characteristics are... " and expected the rest of the sentence to reveal what they are, but instead I think with "the characteristics" you mean the characteristics of the MediaStreamTrack just mentioned. Maybe "it's characteristics"?